.appveyor.yml

```diff
@@ -25,8 +25,8 @@ shallow_clone: false
 
 environment:
   DTS: datalad_next
-  # 20.04 would require a 2.31 GLIBC, but git-annex is only built for 2.34
-  APPVEYOR_BUILD_WORKER_IMAGE: Ubuntu2204
+  APPVEYOR_BUILD_WORKER_IMAGE: Ubuntu2004
```

Comment:
I think there was no good reason other than "it was like this in the beginning and we haven't updated it since" for this older Ubuntu. Thanks for reinstating it, but I think we can keep it more modern and stay with 22.04